core/src/net/sf/openrocket/rocketcomponent/Parachute.java

```diff
@@ -9,24 +9,31 @@
 
 public class Parachute extends RecoveryDevice {
 	private static final Translator trans = Application.getTranslator();
-	
-	public static double DEFAULT_CD = 0.8;
-	
+	private final double defaultDiameter = 0.3;
 	private double diameter;
+	public static double DEFAULT_CD = 0.8;
+	private final Material defaultLineMaterial;
+	private Material lineMaterial;
+	private final int defaultLineCount = 6;
+	private int lineCount = defaultLineCount;
```

Comment:
I would declare this in the Parachute() constructor, so leave this as `private int linecount;` and in `public Parachute()` after `this.diameter = DEFAULT_DIAMTER` add `lineCount = DEFAULT_LINE_COUNT`. It's just a better Java practice

Reply:
Changed.